web/src/lib/components/shared-components/side-bar/side-bar-section.svelte

```diff
@@ -3,6 +3,7 @@
 
 <section
   id="sidebar"
+  tabindex="-1"
```

Comment:
This pulls the sidebar container out of the tab order, so the next tab stop after `<main>` is the sidebar buttons.